staging/src/k8s.io/apiextensions-apiserver/test/integration/conversion/conversion_test.go

```diff
@@ -686,11 +686,17 @@ func expectConversionFailureMessage(id, message string) func(t *testing.T, ctc *
 
 		for _, verb := range []string{"get", "list", "create", "update", "patch", "delete", "deletecollection"} {
 			t.Run(verb, func(t *testing.T) {
+				lastRV := objv1beta2.GetResourceVersion()
 				switch verb {
 				case "get":
-					_, err = clients["v1beta2"].Get(context.TODO(), obj.GetName(), metav1.GetOptions{})
+					var get *unstructured.Unstructured
+					get, err = clients["v1beta2"].Get(context.TODO(), obj.GetName(), metav1.GetOptions{})
+					if get != nil {
+						lastRV = get.GetResourceVersion()
```

Comment:
I don't understand this - it will never be used...

Reply:
Right, confirmed it's not needed.